code/Modules/Props/Types/PhysicalProp.cs

```diff
@@ -13,12 +15,32 @@ public PhysicalProp(PropData data) : base(data)
 	    {
 	    }
 
+	    public PhysicsBody Body => PhysicsBody;
+
 	    public override void Spawn()
         {
             base.Spawn();
+            SetupPhysicsFromModel(PhysicsMotionType.Dynamic);
             PhysicsEnabled = true;
             UsePhysicsCollision = true;
             MoveType = MoveType.Physics;
         }
+
+        public void OnDragStart(MainPlayer player)
+        {
+        }
+
+        public void OnDragStop(MainPlayer player)
+        {
+        }
+
+        public void OnDrag(MainPlayer player)
+        {
+        }
+
+        public bool IsDraggable(MainPlayer player)
+        {
+            return true;
+        }
```

Comment:
suggestion: Duplication of code 
Maybe we can factorize

Reply:
Prop and Item will extends from the same object in the future, I created the ticket